src/development/LocalTest/Controllers/HomeController.cs

```diff
@@ -134,6 +134,38 @@ public async Task<ActionResult> LogInTestUser(StartAppModel startAppModel)
             // Ensure that the documentstorage in LocalTestingStorageBasePath is updated with the most recent app data
             await _applicationRepository.Update(app);
 
+            if(_localPlatformSettings.LocalAppMode == "http")
+            {
+                // Instantiate a prefill if a file attachment exists.
+                var prefill = Request.Form.Files.FirstOrDefault();
+                if (prefill != null)
+                {
+                    var owner = prefill.FileName.Split(".")[0];
```

Comment:
I get the instanceOwner from the filename `{orgnr}.xml` or `{fnr}.xml`.

Any suggestions for better ways to specify this (apart from a free test field that needs to be filled every time.)

Maybe a dropdown for all the orgs "Sophie Salt" has roles in?

Reply:
I think using the file name could be an okay convention in the beginning. It's not going to feel good if you need to frequently change the name of a file, but it could work fine as a start.

I have been wanting to let you login and select reportee as a separate operation from "start app".  LocalTest should as an example be able to list existing instances for selected reportee. That would be a bigger change though.

Reply:
Would be easier if https://github.com/Altinn/altinn-studio/pull/9179 would be considered.